src/util.js

```diff
@@ -4,6 +4,6 @@ export function assign(obj, props) {
 }
 
 export function getKey(vnode, fallback) {
-	let key = vnode.attributes && vnode.attributes.key;
+	var key = vnode.key || (vnode.attributes && vnode.attributes.key);
```

Comment:
vnode.attributes could be removed that references to nothing in Inferno. Your fix is correct here.